release.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh -ex
+
+case $(uname) in
+    Darwin|Linux)
+        sudo=sudo
+        zip=zip
+        ;;
+    *_NT-*)
+        exe=.exe
+        zip=/c/msys64/usr/bin/zip
+        ;;
+    *)
+        echo "unknown OS: $(uname)" >&2
+        exit 1
+        ;;
+esac
+
+#
+# Install Zeek packages.  We don't use zkg because it didn't work
+# out-of-the-box in recent attempts and our package installation
+# requirements are little more than copying scripts.  We already had
+# the approach below working in our prior Windows port so we'll
+# stick with it for now.
+#
+
+zkg_meta() {
+    section=${1:?'section required'}
+    option=${2:?'option required'}
+    python3 <<EOF
+import configparser
+c = configparser.ConfigParser()
+c.read('zkg.meta')
+print(c.get('$section', '$option', fallback=''))
+EOF
+}
+
+install_zeek_package() {
+    github_repo=${1:?'github_repo required'}
+    git_ref=${2:?'git_ref required'}
+    package=${github_repo#*/}
+    mkdir $package
+    (
+        export PATH=/usr/local/zeek/bin:$PATH
+        cd $package
+        curl -sL https://github.com/$github_repo/tarball/$git_ref |
+            tar -xzf - --strip-components 1
+
+        script_dir=$(zkg_meta package script_dir)
+        $sudo cp -r "$script_dir" /usr/local/zeek/share/zeek/site/$package/
+
+        build_command=$(zkg_meta package build_command)
+        if [ "$build_command" ]; then
+            if [ "$OS" = Windows_NT ]; then
+                export LDFLAGS='-static -Wl,--allow-multiple-definition'
+            fi
+            sh -c "$build_command"
+            $sudo tar -xf build/*.tgz -C /usr/local/zeek/lib/zeek/plugins
+        fi
```

Comment:
This and other references to "plugins" are part of what came over from the glue in our older Zeek port. They should be effectively no-ops at this point because we now use the built-in Community ID support and hence don't need to compile the [plugin](https://github.com/corelight/zeek-community-id) like we did in the past. It was a close call in my mind as to whether I should try to remove all the plugin refs from this script and the "zeekrunners" or leave it there in case there's a reason to revive it in the future. Since everything builds and runs fine with what's currently in this PR I figured I'd save myself the extra iterations of removing and testing without it, but if someone feels strongly I can invest the time.